### DISCUSSION TITLE
High School Scaler

### DISCUSSION THREAD

**Discussion by amitmac** [scaleracademy/open-source-projects#82] [Ideas]
#### Overview  
We can build a similar application as Scaler for high school students to make them good in Maths, Science (PCB). It can help students prepare for competitive exams like IIT-JEE, Olympiads, NTSE, etc. This will be similar to BYJU'S or Unacademy. 

#### Idea
Students can take multiple courses in Maths, Science based on their class. There will be multiple chapters on each subject. After each session, there will be an assignment. Students will be ranked based on their scores. If a student faces any difficulty, he/she can raise a help request which can be resolved by instructors, peer students, higher class students, etc. Students can rate each class and give ratings to their instructors. 

We'll host competitions as well in maths and science. We can give prizes to the students who perform well. 

:sparkles: **Welcome to the online Stanford for high school students** :sparkles:

**Comment by MrKrishnaAgarwal** [comment]
Good Idea, can be implemented 👍🏻
